pages/form-builder/support/[[...supportType]].tsx

```diff
@@ -116,6 +122,29 @@ export default function Contactus() {
             required
           />
         </div>
+        <div className="focus-group mt-14">
+          <Label id={"label-department"} htmlFor={"department"} className="required" required>
+            {t("contactus.department")}
+          </Label>
+          <TextInput
+            type={"text"}
+            id={"department"}
+            name={"department"}
+            className="required w-[34rem]"
+          />
+        </div>
+        <div className="focus-group mt-14">
+          <Label id={"label-branch"} htmlFor={"branch"} className="required" required>
+            {t("contactus.branch")}
+          </Label>
+          <TextInput type={"text"} id={"branch"} name={"branch"} className="required w-[34rem]" />
+        </div>
+        <div className="focus-group mt-14">
+          <Label id={"label-title"} htmlFor={"title"} className="required" required>
+            {t("contactus.title")}
```

Comment:
I think this change from `contactus.jobtitle` to this - `contactus.title` has resulted in a bug. It's now showing up as "Contact us" instead of "Job Title"


<img width="598" alt="Screenshot 2023-08-10 at 9 49 30 AM" src="https://github.com/cds-snc/platform-forms-client/assets/916044/ea4ef712-eaa7-4bc2-84b9-f457f8d6817f">

Could we just do (completely up to you @thiessenp-cds )
```suggestion
            {t("contactus.jobTitle")}
```

Reply:
Thanks for taking another look. From the start I went back and forth on title vs jobTitle vs job_title vs job-title. I went with jobtitle because it felt like a middle ground. But your point about consistency makes the most sense